spec/features/forms/form/printable_spec.rb

```diff
@@ -14,26 +14,37 @@
   end
 
   describe "print from index" do
-    scenario "should work and show tip the first time" do
-      visit(forms_path)
+    context "first time" do
+      before do
+        visit(forms_path)
+        page.execute_script("localStorage.removeItem('form_print_format_tips_shown')")
+      end
 
-      # First time printing should show tips.
-      find("a.print-link").click
-      expect(page).to have_css("h4", text: "Print Format Tips")
+      it "should work and show tip" do
+        find("a.print-link").click
+        expect(page).to have_css("h4", text: "Print Format Tips")
+
+        click_button("OK")
+        wait_for_load
+
+        # Should still be on same page.
+        expect(current_url).to end_with("forms")
+      end
     end
 
-    # Couldn't get this spec to work on headless chrome. Maybe it will work later.
-    # context "with shown flag set" do
-    #   scenario "should not show tip" do
-    #     visit(forms_path)
-    #     expect(page).to have_content('Form')
-    #     date = Date.today.strftime("%Y-%m-%d")
-    #     page.execute_script("window.localStorage.setItem('form_print_format_tips_shown', '#{date}')")
-    #
-    #     # Second time printing should not show tips.
-    #     find('a.print-link').click
-    #     expect(page).not_to have_css('h4', text: 'Print Format Tips')
-    #   end
-    # end
+    context "with shown flag set" do
```

Comment:
Hmm, thanks for your energy toward getting this "back from the ☠️"! But it still seems to be failing on CI... Perhaps investigate a bit more but not sink hours into it?